src/shadowbox/README.md

```diff
@@ -1,203 +1,169 @@
-# Outline Server
+# Outline Server (Shadowbox)
 
-The internal name for the Outline server is "Shadowbox". It is a server set up
-that runs a user management API and starts Shadowsocks instances on demand.
+The Outline Server, codenamed "Shadowbox," is designed to streamline the setup and sharing of Shadowsocks servers. It includes a user management API and creates Shadowsocks instances when needed. The [Outline Manager](https://github.com/Jigsaw-Code/outline-apps/) manages the server, and can be used as proxy by the [Outline Client](https://github.com/Jigsaw-Code/outline-apps/) or any standard Shadowsocks client.
```

Comment:
I wouldn't call it a "codename". That sounds like a secret. "internal name" is more appropriate.

Also, "Shadowbox" is the Outline Server Docker Image, not really the Outline Server.

Reply:
This sounds like the Outline Manager can be used as a proxy.

Reply:
Good point. Rephrased.